packaging/systemd/cri-docker.service

```diff
@@ -1,8 +1,7 @@
 [Unit]
 Description=CRI Interface for Docker Application Container Engine
 Documentation=https://docs.mirantis.com
-BindsTo=containerd.service
-After=network-online.target firewalld.service containerd.service docker.service
+After=network-online.target firewalld.service docker.service
```

Comment:
Although we assume that containerd should be running when dockerd is running. Here I prefer to keep `containerd.service`

Reply:
It might be running anyway, that is entirelly up to docker... Older versions had docker-containerd running.

This CRI plugin should not bother with how docker runs, since it only needs to talk to the docker daemon ?